<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="nounList">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>260</y>
      <width>101</width>
      <height>192</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Word One</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Two</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Three</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Four</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Five</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Six</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Seven</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Eight</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Nine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Ten</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="headlineList">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>121</width>
      <height>192</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Headline One</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Headline Two</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Headline Three</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Headline Four</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Headline Five</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Headline Six</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Headline Seven</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Headline Eight</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Headline Nine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Headline Ten</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="adjectiveList">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>260</y>
      <width>101</width>
      <height>192</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Word One</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Two</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Three</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Four</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Five</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Six</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Seven</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Eight</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Nine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Ten</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="verbs">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Verbs</string>
    </property>
   </widget>
   <widget class="QLabel" name="headlineLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select any headline!</string>
    </property>
   </widget>
   <widget class="QListWidget" name="verbList">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>101</width>
      <height>192</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Word One</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Two</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Three</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Four</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Five</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Six</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Seven</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Eight</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Nine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Ten</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="nouns">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouns</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adjectives">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Adjectives</string>
    </property>
   </widget>
   <widget class="QPushButton" name="media">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Media</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
